homepage.html

```diff
@@ -0,0 +1,186 @@
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#log-in-fields">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#">Danebook</a>
+        </div>
+
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="log-in-fields">
+          <form class="navbar-form navbar-right">
+            <div class="form-group">
+              <input type="email" class="form-control" placeholder="Email">
+              <input type="password" class="form-control" placeholder="Password">
+            </div>
+            <button type="submit" class="btn btn-default">LOG IN</button>
+          </form>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <main class="container">
+      <div class="row">
+        <div class="home-page-left col-sm-6">
+          <h3>Connect with all your friends!</h3>
+          <ul class="things-to-do-with-friends">
+            <li>See photos and update your news feed.</li>
+            <li>Post your status for the world to see from your profile.</li>
+            <li>Get in touch with friends by "friending" them.</li>
+            <li>Like things because you're a positive person!</li>
+          </ul>
+        </div>
+
+        <div class="sign-up-form col-sm-5">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>Sign up</h1>
+            </div>
+          </div>
+          <form>
+            <div class="row">
+              <div class="col-md-6" id="first-name">
+                <input class="textfield" type="text" placeholder="First name">
+              </div>
+              <div class="col-md-6">
+                <input class="textfield" type="text" placeholder="Last name">
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-md-12">
```

Comment:
12 width column might not be appropriate for a text field that doesnt take up the entire width of the row